resources/js/src/app/components/itemList/filter/FindologicItemFilter.spec.ts

```diff
@@ -0,0 +1,351 @@
+import Vuex from 'vuex';
```

Comment:
Running `npm run lint` will show several linting issues. Please try to resolve them.